src/dashboard/src/components/administration/views_processing.py

```diff
@@ -64,6 +72,57 @@ def write_to_file(self, file_path):
         file = open(file_path, 'w')
         file.write(etree.tostring(self.xml, pretty_print=True))
 
+
+def save__normalize(name, value, choice_uuid, do_not_lookup, xml_choices, default_processing_func):
+    """
+    There are two "Normalize" chain links with choices "Do not normalize", refs #10216.
+    """
+    if value != 'Do not normalize':
+        default_processing_func(value, choice_uuid, do_not_lookup, xml_choices)
+        return
+    sql = """
+        SELECT
+            MicroServiceChainLinks.pk,
+            MicroServiceChains.pk
+        FROM TasksConfigs
+        LEFT JOIN MicroServiceChainLinks ON (MicroServiceChainLinks.currentTask = TasksConfigs.pk)
+        LEFT JOIN MicroServiceChainChoice ON (MicroServiceChainChoice.choiceAvailableAtLink = MicroServiceChainLinks.pk)
+        LEFT JOIN MicroServiceChains ON (MicroServiceChains.pk = MicroServiceChainChoice.chainAvailable)
+        WHERE
+            TasksConfigs.description = 'Normalize'
+            AND MicroServiceChains.description = 'Do not normalize';
```

Comment:
Would be nice if this could be done with the ORM, instead of raw SQL, but I'm not sure there's an easy way to do that since you need paired values from 2 tables.